README.rst

```diff
@@ -27,13 +27,31 @@
 ------
 
 
-A simple web scraping tool for recipe sites.
+A simple scraping tool for recipe webpages.
+
+
+Netiquette
+----------
+
+If you're using this library to collect large numbers of recipes from the web, please use the software responsibly and try to avoid creating high volumes of network traffic.
+
+Python's standard library provides a ``robots.txt`` `parser <https://docs.python.org/3/library/urllib.robotparser.html>`_ that may be helpful to automatically follow common instructions specified by websites for web crawlers.
+
+Another parser option -- particularly if you find that many web requests from ``urllib.robotparser`` are blocked -- is the `robotexclusionrulesparser <https://pypi.org/project/robotexclusionrulesparser/>`_ library.
+
+
+Getting Started
+---------------
+
+Start by using `Python's built-in package installer <https://docs.python.org/3/installing/index.html>`_, ``pip``, to install the library:
 
 .. code:: shell
 
-    pip install recipe-scrapers
+    python -m pip install recipe-scrapers
+
+This should produce output about the installation process, and the final line should read: ``Successfully installed recipe-scrapers-<version-number>``.
 
-then:
+Once the library is installed, run ``python`` at the command-line to open an interpreter session, and then begin typing -- and/or modifying -- the code below to scrape some recipes!
```

Comment:
Is it necessary to give instructions on how to open the interpreter? Someone could use the same code in a Python file. If we wanted to keep it specific to the interpreter, should we use the notation that shows the expected output from each line (using `>>>`, as shown briefly here https://docs.python.org/3/tutorial/interpreter.html)?

Reply:
Thanks for the review!  Yep, that's a good point that the code could alternatively be run from a file.  I'll try to make it seem like less of a requirement, and more of a "you could run this interactively to find out how it works".

I'd been trying to decide how/whether to place the expected prompt (`>>>`) into the intro - perhaps hyperlinking out to that tutorial could help with that?  I'll try that approach soon.

Reply:
I'm not sure it's perfect -- and I used some placeholder Python interpreter version output so that it should remain approximately the same level of inaccuracy and irrelevance from day one to day n without requiring updates or appearing unexpected dated -- but I think that this is improved now.  Glad to find out what you think.